docs/devops-guidelines/Create.md

```diff
@@ -0,0 +1,66 @@
+|Version Control|All artifacts are versioned and revisions to those artifacts are tracked by a version control system.  Merges are simple and reversion to a "known-good state" is easy.  The version control system supports distributed development and forms the backbone for the build/deployment pipelines.|git based Azure DevOps Repos |
+
+# Design
+Designs may consist of wireframes, mockups or prototypes. A wireframe is a low-fidelity visual representation of the site and will be used to depict the basic UI, detailing the layout and structure of the site. A mockup is a mid to high-fidelity visual representation of the site and will be used to highlight color scheme, visual style and typography. A prototype is a high-fidelity visual representation of the site and will be used to simulate user interaction by allowing the user to experience the content through clickable elements and interactions. In the event the design process needs to be accelerated, wireframes may be substituted with mockups.
+
+Designs will utilize Material Design, Font Awesome Icons and custom components. Phillips 66 brand guidelines will influence design concept and style. 
+
+Designs will be reviewed by the design team, development team and business analysts before handing off for development. User stories, acceptance criteria and UX personas are used to approve and validate designs. Optionally, user testing may be included in design review. 
+
+# Implementation
+Implementation is where the rubber meets the road.  The plans are laid.  The user stories are written.  The test cases have been identified and it's finally time to build something.
+
+Implementation will follow the SCRUM methodology and utilize Test Driven Development (TDD).  All features will be logged as user stories with acceptance criteria before they are worked.
+
+All development is expected to be performed using Visual Studio Code, Mulesoft Anypoint Studio and Eclipse.
+
+## Source Management
+
+Git repos in Azure DevOps are preferred. 
+
+Associate check ins using the #<work item number> in comments with Work Items to automatically generate release information as well as generate an audit trail
+
+All code is to be checked into a version control system on a regular basis.  More frequent check-ins provide benefits:
+- Team members can synchornize changes reducing efforts to code merge and staying up to date on changes
+- Continous integration builds can be triggered to verify code
+
+[Learning git](https://docs.microsoft.com/en-us/azure/devops/learn/git/what-is-git)
+
+
+## Coding Standards
```

Comment:
Recommend removing this section as I think it is handled by our own coding guidelines pages.